app/src/main/java/org/breezyweather/main/fragments/ManagementFragment.kt

```diff
@@ -278,7 +291,7 @@ open class ManagementFragment : MainModuleFragment(), TouchReactor {
                     }
                     AndroidView(
                         modifier = Modifier
-                            .fillMaxHeight()
+                            .fillMaxSize()
```

Comment:
This fixes an issue where the items (locations) in the location list aren't consuming the entire width in landscape mode when the navigation bar is hidden ([issue](https://github.com/user-attachments/assets/49871cf9-d0c9-4401-900f-04fab15794d9), [fix](https://github.com/user-attachments/assets/d2f717b6-0e43-4562-8712-02cf4ce10b03)).